index.html

```diff
@@ -175,6 +225,37 @@ <h5 class="about-h5">Skills</h5>
           </li>
         </ul>
       </section>
+      <!-- form-section -->
+      <section class="form-section">
+        <h3 class="contact-me">Contact me</h3>
+        <p class="apply">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mqknzyow" method="POST">
+          <input
+            type="text"
+            name="user_name"
+            placeholder="User name"
+            required
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="user_email"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            placeholder="Write your message here"
+            name="user_text"
+            required
+            maxlength="500"
+          ></textarea>
+          <button type="submit">Get It Touch</button>
+        </form>
+      </section>
```

Comment:
- [OPTIONAL] Kindly wrap this section separately into a `footer` tag out of the general `section` tag to make your code more readable, understandable and more sematic👌